app/views/catalog/_facet_limit.html.erb

```diff
@@ -1,4 +1,9 @@
 <% # Overridden from Blacklight to use   %>
+<% if article_search? && facet_field.key == 'eds_publication_type_facet'%>
+  <div class="alert alert-warning" role="alert">
+    <%= t('searchworks.articles.flashes.source_type_warning').html_safe %>
```

Comment:
What do you think about renaming the key to use an `_html` suffix? I _think_ this may allow us to not have to call `html_safe` on it.

Reply:
neat, that worked! Thanks for the i18n ✨magic✨ @mejackreed!